sbnobj/Common/POTAccounting/classes_def.xml

```diff
@@ -0,0 +1,14 @@
+<lcgdict>
+  <class name="sbn::BNBSpillInfo" ClassVersion="15">
+   <version ClassVersion="15" checksum="1226979720"/>
+   <version ClassVersion="14" checksum="2089752727"/>
+   <version ClassVersion="13" checksum="1538708656"/>
+   <version ClassVersion="12" checksum="2483311428"/>
+   <version ClassVersion="11" checksum="4133298356"/>
+   <version ClassVersion="10" checksum="13541"/>
```

Comment:
Remove the development versions and make this be version `10`.

Reply:
I tried to do this but I think I failed...